Update.ArreglosHorizontal

Spinerati opened the pull request:
Se arreglaron varios errores de la escena horizontal, ahora el jefe solo colisiona al aparecer, el juego no crashea

Tarifa17 commented:
Se arreglaron varios errores de la escena horizontal, ahora el jefe solo colisiona al aparecer, el juego no crashea

Spinerati committed:
Update.ArreglosHorizontal

Se arreglaron varios errores de la escena horizontal, ahora el jefe solo colisiona al aparecer, el juego no crashea
